tests/CMakeLists.txt

```diff
@@ -150,8 +150,9 @@ set_property(TEST ship_if_test PROPERTY LABELS nonparallelizable_tests)
 
 add_test(NAME ship_streamer_test COMMAND tests/ship_streamer_test.py -v --num-clients 10 ${UNSHARE} WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 set_property(TEST ship_streamer_test PROPERTY LABELS long_running_tests)
-add_test(NAME ship_streamer_if_test COMMAND tests/ship_streamer_test.py -v --num-clients 10 --activate-if ${UNSHARE} WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
-set_property(TEST ship_streamer_if_test PROPERTY LABELS long_running_tests)
+# uncomment after https://github.com/AntelopeIO/leap/issues/2285 implemented
```

Comment:
Add a note in 2285 to uncomment this so we won't forget.

Reply:
already done :-)!